newtmgr/cli/res.go

```diff
@@ -92,6 +93,39 @@ func extractResKv(params []string) (map[string]interface{}, error) {
 	return m, nil
 }
 
+/* Helper functions to convert JSON object into pretty format
+   Adapted from elastic/beats/libbeat/common/mapstr.go
+*/
+
+func cleanUpInterfaceArray(in []interface{}) []interface{} {
+	result := make(map[string]interface{})
+	for k, v := range in {
+		result[i] = cleanUpMapValue(v)
+	}
+	return result
```

Comment:
newtmgr/cli/res.go:105:2: cannot use result (type map[string]interface {}) as type []interface {} in return argument